azext_iot/tests/central/__init__.py

```diff
@@ -58,9 +55,13 @@ class CentralLiveScenarioTest(CaptureOutputLiveScenarioTest):
     def __init__(self, test_scenario):
         super(CentralLiveScenarioTest, self).__init__(test_scenario)
         self._create_app()
```

Comment:
self._create_app() may set variables like the token, dns_suffix and the app primary key , however later in initialization (like 65, 66) self.token and self.dns_suffix get set to `None`, preventing the respective test configuration.

CC: @gtrifonov

Reply:
fixe - going to fix the description for the functions too